lib/fluent/plugin/in_tail.rb

```diff
@@ -594,6 +596,9 @@ def flush_buffer(tw, buf)
 
     # @return true if no error or unrecoverable error happens in emit action. false if got BufferOverflowError
     def receive_lines(lines, tail_watcher)
+      lines = lines.reject do |line|
+        @skip_long_lines ? line.bytesize > @skip_long_lines : false
```

Comment:
Should output warning log to notify that some lines are skipped, but shouldn't output the line itself to avoid filling up the log.
It would be better that outputting the line in debug or trace log.

Reply:
If I was a system administator, I would _really_ like to know when any record skip happened.

I believe adding `log.warn("warning: ")` (or `log.info`?) to this failure path will be a great idea.
For reference, here is how Fluent Bit handles it:

https://github.com/fluent/fluent-bit/blob/4725fb24e677181e548c6cbeeb7c7ad59224d0ec/plugins/in_tail/tail_file.c#L1161

Reply:
It is reasonable to do so.

Reply:
Fixed.